docs/pvcscale.md

```diff
@@ -0,0 +1,179 @@
+# PVCScale Workload
+
+The pvcscale workload playbook is `workloads/pvcscale.yml` and will run the pvcscale workload on your cluster.
+
+PVCScale workload is designed to stress storage backend from the scale point of
+view. It will create desired number of pods and attach an PVC to each pod.
```

Comment:
Perhaps: "It creates a desired number of pods and attaches a PVC to each pod."?

Reply:
can be like this or something else :), will rework this part anyway.